data-protocols/ids/ids-api-multipart-endpoint-v1/src/test/java/org/eclipse/dataspaceconnector/ids/api/multipart/handler/ArtifactRequestHandlerTest.java

```diff
@@ -159,4 +163,13 @@ private MultipartRequest createMultipartRequest(DataAddress dataDestination, Str
                 .claimToken(claimToken)
                 .build();
     }
+
+    private ObjectMapper getCustomizedObjectMapper() {
+        var typeManager = new TypeManager();
+        typeManager.registerContext("ids", JsonLd.getObjectMapper());
+
+        TypeManagerUtil.registerIdsClasses(typeManager);
```

Comment:
Since these two lines (registerContext and registerIdsClasses) are called a lot of times one next to the other, it would be good to encapsulate it into the `TypeManagerUtil` (that could become `IdsTypeManagerUtil`)